projects/ids-enterprise-ng/src/lib/accordion/soho-accordion.component.ts

```diff
@@ -147,6 +162,10 @@ export class SohoAccordionComponent implements AfterViewInit, OnDestroy {
    */
   @Input() public set hasPanels(bool: boolean) {
     this.options.hasPanels = bool;
+    if (this.accordion) {
+      this.accordion.settings.allowOnePane = this.options.allowOnePane;
```

Comment:
Should this be hasPanels (but thats not a setting). Maybe it should be allowOnePane?